PinClustering/PinClustering/ViewControllers/TownsViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  TownsViewController.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 02/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import UIKit
+
+class TownsViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private static let cellId = "townCellIdentifier"
+    private static let webServiceAddress = "https://data.cityofchicago.org/resource/alternative-fuel-locations.json?"
+    private static let webServiceDescription = "Alternative Fuel Locations"
+    private static let townName = "Chicago"
+    private static let rowHeight: CGFloat = 70.0
+
+    @IBOutlet weak var townsTableView: UITableView!
+    
+    private var towns = [Town]()
+    
+    // MARK: Lifecycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        setupTownsTableView()
+        loadTowns()
+    }
+    
+    override func viewWillAppear(animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.navigationBarHidden = true
+    }
+    
+    // MARK: Private Methods
+    
+    func setupTownsTableView() {
```

Comment:
i don't see the point to create a method for one line of code. Why don't you just put it in the viewDidLoad?